input/pagecontent/changes.md

```diff
@@ -170,11 +170,15 @@ This change log documents the significant updates and resolutions implemented fr
   - removed the required binding to ObservationStatus Result Available value set from Observation.status [FHIR-45125](https://jira.hl7.org/browse/FHIR-45125)
   - updated invariant au-core-obs-01 to change the length check from >= 10 to >= 8 to match required precision to the day [FHIR-46407](https://jira.hl7.org/browse/FHIR-46407)
   - removed Obligations on Observation.hasMember.reference [FHIR-46728](https://jira.hl7.org/browse/FHIR-46728)
+  - removed the cardinality constraint on Observation.hasMember.reference, changing it from 1..1 to 0..1 [FHIR-48331](https://jira.hl7.org/browse/FHIR-48331)
```

Comment:
Merge the two change entries for the same element and summarise, basically we removed constraints.

Reply:
Merged, thanks